app/src/main/java/org/mozilla/fenix/tabstray/ext/BrowserMenu.kt

```diff
@@ -18,7 +18,7 @@ fun BrowserMenu.showWithTheme(view: View) {
         (popupMenu.contentView as? CardView)?.setCardBackgroundColor(
             ContextCompat.getColor(
                 view.context,
-                R.color.fx_mobile_layer_color_1,
+                R.color.fx_mobile_layer_color_2,
```

Comment:
I assume there are other Dropdown Menu where we will need to align with this new color if we were to go towards this direction.

Reply:
I think I am mistaken about my previous comment. I see now from actually testing things that the other DropdownMenu is already using Layer2.

Reply:
Can confirm the dropdown menu should user Layer2. Thanks!